<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeakerWidget</class>
 <widget class="QWidget" name="SpeakerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>131</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
	font: bold 10pt;
    color: #2E2E2E;
	shadow: ;
}

QFrame#frame {
    border-top: 2px solid rgba(241,240,230,255);
    border-bottom: 2px solid rgba(171,162,153,255);
    border-right: 2px solid rgba(157,152,145,255);
    border-left: 2px solid rgba(218,206,196,255);
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0.006, y2:0, stop:0 rgba(214, 203, 189, 255), stop:1 rgba(238, 235, 235, 255));
}

QSlider {
    padding: 0;
    margin: 0;
}

QSlider::groove:vertical {
     position: absolute; /*  absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */
     left: 0; right: 0; top: 0; bottom: 0;
     width: 20px;
	 background-image: url(:/gui/volume-slider-bg);
 }

 QSlider::handle:vertical {
     height: 18px;
     background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(128, 128, 128, 255), stop:0.423729 rgba(237, 237, 237, 255), stop:0.740113 rgba(237, 237, 237, 255), stop:1 rgba(140, 140, 140, 255));
     margin: 0 6px; /* expand outside the groove */
     border-top: 1px solid rgba(236,236,236,255);
     border-radius: 2px;
 }

QSlider#volumeSlider::add-page:vertical {
	background-image: url(:/gui/volume-slider-sub-bg);
}

QFrame#volumeFrame {
    border-radius: 3px;
    border: 0;
    background: rgba(14,64,112,255);
    margin: 5px 0;
    width: 53px;
    min-width: 53px;
}

QProgressBar {
    border: 0;
	background-image: url(:/gui/volume-slider-bg);
}

QProgressBar::chunk {
	background-image: url(:/gui/volume-slider-sub-bg);
    margin-top: 0.5px;
    height: 5px;
}

QPushButton#acceptButton,
QPushButton#acceptButton:checked,
QPushButton#acceptButton:pressed,
QPushButton#dismissButton,
QPushButton#dismissButton:checked,
QPushButton#dismissButton:pressed {
    border: 0;
    background: none;
    padding: 0;
}

QLineEdit {
    padding: 0;
    margin: 0;
    padding-left: 3px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLineEdit" name="delayBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0 min</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="companyLabel">
          <property name="text">
           <string>Company</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="volumeFrame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QSlider" name="volumeSlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="volumeBar">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="textDirection">
            <enum>QProgressBar::TopToBottom</enum>
           </property>
           <property name="format">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="acceptButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/gui/accept-button</normaloff>:/gui/accept-button</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dismissButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/gui/desmiss-button</normaloff>
          <normalon>:/gui/dismiss-button-ressed</normalon>:/gui/desmiss-button</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
